options/locale/locale_en-US.ini

```diff
@@ -1990,6 +1990,8 @@ settings.event_package = Package
 settings.event_package_desc = Package created or deleted in a repository.
 settings.branch_filter = Branch filter
 settings.branch_filter_desc = Branch whitelist for push, branch creation and branch deletion events, specified as glob pattern. If empty or <code>*</code>, events for all branches are reported. See <a href="https://pkg.go.dev/github.com/gobwas/glob#Compile">github.com/gobwas/glob</a> documentation for syntax. Examples: <code>master</code>, <code>{master,release*}</code>.
+settings.authorization_header = Authorization Header
+settings.authorization_header_desc = "If it's not empty, it will be included as an authorization header to the webhook request. Examples: <code>Bearer token123456</code>, <code>Basic YWxhZGRpbjpvcGVuc2VzYW1l</code>."
```

Comment:
Can we move examples out of the translation, e.g. end in `Examples:`?

Reply:
I edited the code to use string substitution: `Examples: %s.`

Is it what you had in mind?

Reply:
Yeah that'll work as well and will be slightly more friendly to translators :)